.drone.yml

```diff
@@ -5,7 +5,8 @@ name: rbabylon-40
 #Gobal variable definitions and anchors
 global-variables:
     environment: &default_environment
-        BABYLON_URL: https://www.github.com/metrumresearchgroup/babylon/releases/latest/download/bbi_linux_amd64.tar.gz
+        #BABYLON_URL: https://www.github.com/metrumresearchgroup/babylon/releases/latest/download/bbi_linux_amd64.tar.gz
```

Comment:
Do we want to leave in commented-out code?

Reply:
Good catch. This is all related to the issue mentioned here: https://github.com/metrumresearchgroup/rbabylon/pull/95#discussion_r442441634

When the babylon release has been made (before we merge) we will switch these back.